templates/frontend/objects/monograph_summary.tpl

```diff
@@ -8,11 +8,17 @@
  * @brief Display a summary view of a monograph for display in lists
  *
  * @uses $monograph Monograph The monograph to be displayed
+ * @uses $isFeatured bool Is this a featured monograph?
  *}
-<div class="obj_monograph_summary">
+<div class="obj_monograph_summary{if $isFeatured} is_featured{/if}">
 	<a href="{url page="catalog" op="book" path=$monograph->getId()}" class="cover">
 		<img alt="{translate key="catalog.coverImageTitle" monographTitle=$monograph->getLocalizedFullTitle()|strip_tags|escape}" src="{url router=$smarty.const.ROUTE_COMPONENT component="submission.CoverHandler" op="thumbnail" submissionId=$monograph->getId() random=$monograph->getId()|uniqid}" />
 	</a>
+	{* {if $isFeatured}
+		<div class="featured">
+			Featured
+		</div>
+	{/if} *}
 	<a href="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="book" path=$monograph->getId()}" class="title">
 		{$monograph->getLocalizedFullTitle()}
```

Comment:
This also needs `|escape`. It's probably worth going through the front-end templates in general.